### PR TITLE
Created Activity Feed Page

### PR DESCRIPTION
**Pull request by Mariamcm90**
Added a page for the activity feed. It currently shows when users send a message in a chat, although I am still figuring out how to link the chat title in the event to the chat's page. I will do that in the next pull request. Marie and I are working on how to integrate other event types into the activity feed. 

Also, I'm not sure why web.xml was indented but the linter changed it when I reformatted the code.

### CODE REVIEW

File: src/main/webapp/WEB-INF/view/activityFeed.jsp [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+          for (Message message : messages) {
+            UUID userId = message.getAuthorId();
+            String userName = UserStore.getInstance().getUser(userId).getName();
+            String content = message.getContent();
+            UUID conversationId = message.getConversationId();
+            String conversationName = ConversationStore.getInstance().getConversationWithId(conversationId).getTitle();
+
+            Instant creationTime = message.getCreationTime();
+            LocalDateTime ldt = LocalDateTime.ofInstant(creationTime, ZoneId.systemDefault());
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MM/dd/yy h:mm:ss a");
```
Comment (kkane098) [inline src/main/webapp/WEB-INF/view/activityFeed.jsp]: We should probably be consistent with how we format dates across the app, so I would either change how they're formatted in profile.jsp to match how they're formatted here or vice versa (doesn't matter which to me).

Comment (Mariamcm90) [reply src/main/webapp/WEB-INF/view/activityFeed.jsp]: You're right. Thanks! I ended up changing the one in profile.jsp.

File: src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -15,84 +15,95 @@
   limitations under the License.
 -->
 <web-app
```
Comment (fatlotus) [inline src/main/webapp/WEB-INF/web.xml]: Should we decide on a single xml style?

**Comment by Mariamcm90** [comment]
Sorry about that! Changed it back.

**Pull request merged**